.github/workflows/network-explorer.yml

```diff
@@ -22,6 +22,8 @@ jobs:
         node-version: '14'
```

Comment:
Is there any reason why we're using node v14 on the workflow?

Reply:
No reason, it could be changed to Node 16 some time